packages/remediations/src/utils/utils.js

```diff
@@ -155,17 +142,28 @@ export const submitRemediation = (formValues, data, basePath) => {
                 ...(formValues[SYSTEMS][id] || [])
             ])
         });}).filter(issue => issue.systems.length > 0);
+
+    const interval = setInterval(() => {
+        percent < 99 && setState({ percent: ++percent });
+    }, (issues.length + Object.keys(formValues[SYSTEMS]).length) / 10);
+
     const add = { issues, systems: [] };
-    if (formValues[EXISTING_PLAYBOOK_SELECTED]) {
-        const { id, name } = formValues[EXISTING_PLAYBOOK];
-        api.patchRemediation(id, { add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)
-        .then(() => resolver(id, name, false, data.onRemediationCreated))
-        .catch(() => resolver(null, name, false, data.onRemediationCreated, true));
-    } else {
-        api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)
-        .then(({ id }) => resolver(id, formValues[SELECT_PLAYBOOK], true, data.onRemediationCreated))
-        .catch(() => resolver(null, formValues[SELECT_PLAYBOOK], true, data.onRemediationCreated, true));
-    }
+
+    const { id } = formValues[EXISTING_PLAYBOOK] || {};
+    const isUpdate = formValues[EXISTING_PLAYBOOK_SELECTED];
+
+    (isUpdate &&
+    api.patchRemediation(id, { add, auto_reboot: formValues[AUTO_REBOOT] }, basePath) ||
+    api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)).then(({ id }) => {
+        setState({ id, percent: 100 });
+        /*data.onRemediationCreated({
```

Comment:
Can we remove this comment?

Reply:
Thanks @Hyperkid123 for catching that, it was a debug comment, I'll remove it.